lib/satellite/warden_strategy.rb

```diff
@@ -30,12 +30,20 @@ def cookies
       env['action_dispatch.cookies']
     end
 
+    def user_cookie
+      @user_cookie ||= Satellite::UserCookie.new(cookies)
+    end
+
+    def jwt_user
```

Comment:
I don't know, the `jwt_cookie` is `user_cookie` already. `@jwt_user` gets assigned an instance of `JWTUserDecoder`, which basically wraps user attributes.